app/assets/stylesheets/views/_homepage.scss

```diff
@@ -1,65 +1,46 @@
-.home-top {
-  overflow: hidden;
-  background: $govuk-brand-colour;
-  color: govuk-colour("white");
-}
+.homepage-inverse-header {
+  background-color: $govuk-brand-colour;
+  padding-bottom: govuk-spacing(7);
+  padding-top: govuk-spacing(8);
 
-.home-top__inner {
-  position: relative;
-}
-
-.home-top__intro {
-  @include govuk-font(19);
-  margin: 0;
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: govuk-spacing(8);
+    padding-top: govuk-spacing(9);
+  }
 }
 
-.home-top__intro--simpler {
+.homepage-inverse-header__title {
+  color: govuk-colour("white");
+  font-size: 32px;
```

Comment:
I get that this is a fallback, but is it necessary if it's IE9 and 10 that we're worried about as they do have partial support for rems? [It's only if we used the `font` shorthand or on pseudo elements where it doesn't work from what I gather](https://caniuse.com/?search=rem).

Reply:
I think the fallback should be kept in because:

 - this is what the [guidance in the GDS way advises to do][1]
 - the support is very good, but not complete - about 2.6% of visitors to GOV.UK in the last month used a browser that didn't support `rem` units

As [Frontend doesn't  `govuk_template` any more][2] we don't need to wrap this in a conditional statement either, which is nice.

[1]: https://gds-way.cloudapps.digital/manuals/programming-languages/css.html#include-fallbacks-for-rem-spacing
[2]: https://github.com/alphagov/frontend/blob/fd8de1514f2973115fa5a89c188a17e5380a8011/app/assets/stylesheets/application.scss#L1

Reply:
Cool thanks! Didn't know it's in GDS Way - I'll double check it.